README.md

```diff
@@ -522,6 +522,7 @@ A list of **Free** and **Open Source Software** ***(FOSS)*** for **Android** –
 ### • Music & Audio Player
 
 * [**Auxio**](https://github.com/OxygenCobalt/Auxio) <sup>**[[F-Droid](https://f-droid.org/app/org.oxycblt.auxio)]**</sup>
+* [**Bloomee🌸**](https://github.com/HemantKArya/BloomeeTunes) <sup>**[[SourceForge](https://bloomee.sourceforge.io)]**</sup>
```

Comment:
(Until now) please don't use emojis to stay consistent with the other entries. Also , the `sup` part is for F-Droid / Izzy only.

You should consider a submission btw. :smiley:.

Reply:
You don't have to :-). Remove Emoji and the `<sup>…</sup>` and this PR is ready to merge.

F-Droid / Izzy is fully optional.

Reply:
Okay that sounds good too.